sirepo/tornado.py

```diff
@@ -24,7 +24,7 @@ async def get(self):
             # on the queue and before this task finishes the await.
             x = super().get()
             return await x
-        except asyncio.CancelledError:
+        except (asyncio.CancelledError, concurrent.futures._base.CancelledError):
```

Comment:
It is unclear to me when we need to handle both of these types of cancelledErrors. It looks like CPython [wraps all concurrent.futures._base.CancelledError's in an asyncio.CancelledError](https://github.com/python/cpython/blob/568fb0ff4aa641329261cdde20795b0aa9278175/Lib/asyncio/tasks.py#L245) for tasks. Tornado doesn't seem to provide the same guarantees. So maybe anytime we are interacting with a Tornado primitive that could be cancelled we need to catch both? Since we are using tornado throughout I think that means everywhere. Agree?